src/components/intro/index.tsx

```diff
@@ -130,7 +135,14 @@ export default class Intro extends Component<Props, State> {
 
     // Wait for the user to accept or dismiss the install prompt
     const { outcome } = await beforeInstallEvent.userChoice;
-    ga('send', 'event', 'pwa-install', installButtonSource, outcome);
+    // Send the analytics data
+    const gaEventInfo = {
+      eventCategory: 'pwa-install',
+      eventAction: 'promo-clicked',
+      eventLabel: installButtonSource,
+      eventValue: outcome === 'accepted' ? 1 : 0,
+    };
+    ga('send', 'event', gaEventInfo);
```

Comment:
This allows you to track several different install promos (each would be identified by `installButtonSource`, and uses the `value` to track if the user accepted or dismissed the install dialog (CTR). Before, we were putting the outcome in the label, which required looking at analytics and doing some manual calculations to get the CTR.